valheimconfig.json

```diff
@@ -94,5 +87,4652 @@
+    "IsFlagArgument":false,
+    "ParamFieldName":"Inventory.karveInventoryRows",
+    "IncludeInCommandLine":false,
+    "DefaultValue":"2",
+    "EnumValues":{},
+    "Suffix": "rows"
+  },{
+    "DisplayName":"Longboat Inventory Columns",
+    "Category":"Valheim Plus - Basic Settings",
+    "Description":"Longboat inventory number of columns. Default 8, Minimum 6, Maximum 8",
+    "Keywords":"Inventory,Longboat,Inventory,Columns",
+    "FieldName":"Inventory.longboatInventoryColumns",
+    "InputType":"number",
+    "MinValue":"6",
+    "MaxValue":"8",
+    "IsFlagArgument":false,
+    "ParamFieldName":"Inventory.longboatInventoryColumns",
+    "IncludeInCommandLine":false,
+    "DefaultValue":"8",
+    "EnumValues":{},
+    "Suffix": "columns"
+  },{
+    "DisplayName":"Longboat Inventory Rows",
+    "Category":"Valheim Plus - Basic Settings",
+    "Description":"Longboat inventory number of rows, more than 4 rows will add a scrollbar. Default 3, Minimum 3, Maximum 30",
+    "Keywords":"Inventory,Longboat,Inventory,Rows",
+    "FieldName":"Inventory.longboatInventoryRows",
+    "InputType":"number",
+    "MinValue":"3",
+    "MaxValue":"30",
+    "IsFlagArgument":false,
+    "ParamFieldName":"Inventory.longboatInventoryRows",
+    "IncludeInCommandLine":false,
+    "DefaultValue":"3",
+    "EnumValues":{},
+    "Suffix": "rows"
+  },{
+    "DisplayName":"Inventory Fill Top To Bottom",
+    "Category":"Valheim Plus - Basic Settings",
+    "Description":"By default tools and weapons go into inventories top to bottom and other materials bottom to top. Enable to make all items go into the inventory top to bottom.",
+    "Keywords":"Inventory,Fill,Top,Bottom",
+    "FieldName":"Inventory.inventoryFillTopToBottom",
+    "InputType":"checkbox",
+    "IsFlagArgument":false,
+    "ParamFieldName":"Inventory.inventoryFillTopToBottom",
+    "IncludeInCommandLine":false,
+    "DefaultValue":"false",
+    "EnumValues":{
+      "False":"false",
+      "True":"true"
+    }
+  },{
+    "DisplayName":"Merge With Existing Stacks",
+    "Category":"Valheim Plus - Basic Settings",
+    "Description":"By default items go to their original position when picking up your tombstone. Enable to make all stacks try to merge with an existing stack first.",
+    "Keywords":"Inventory,Merge,Existing,Stacks",
+    "FieldName":"Inventory.mergeWithExistingStacks",
+    "InputType":"checkbox",
+    "IsFlagArgument":false,
+    "ParamFieldName":"Inventory.mergeWithExistingStacks",
+    "IncludeInCommandLine":false,
+    "DefaultValue":"false",
+    "EnumValues":{
+      "False":"false",
+      "True":"true"
+    }
+  },{
+    "DisplayName":"Items Settings Enable",
+    "Category":"Valheim Plus - Basic Settings",
+    "Description":"Enabling this section gives you access to the settings up to the next section.",
+    "Keywords":"Items,Enable",
+    "FieldName":"Items.enable",
+    "InputType":"checkbox",
+    "IsFlagArgument":false,
+    "ParamFieldName":"Items.enable",

```

Comment:
These should all be `...enabled` with a 'd" on the end. As it is, the settings do not update.